definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -136,24 +110,24 @@
   {
     "LiveFrom": "12/08/2020",
     "CaseTypeID": "DIVORCE",
-    "ID": "PBAReferenceLabel",
-    "Label": "Your pay account reference: ${PBAReference}",
+    "ID": "LabelPBAReference",
+    "Label": "Your pay account reference: ${FeeAccountReference}",
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "12/08/2020",
     "CaseTypeID": "DIVORCE",
-    "ID": "PBANumberLabel",
-    "Label": "Fee account number: ${PBANumber}",
+    "ID": "LabelPBANumber",
+    "Label": "Fee account number: ${SolicitorFeeAccountNumber}",
```

Comment:
This field also already exists: LabelSolicitorFeeAccountNumber
<img width="555" alt="Screenshot 2020-08-17 at 11 50 58" src="https://user-images.githubusercontent.com/16866068/90388597-f68b1580-e07f-11ea-802e-99c15109c962.png">

Reply:
Just for solicitor actually